easybuild/framework/easyblock.py

```diff
@@ -2799,7 +2799,10 @@ def update_config_template_run_step(self):
         """Update the the easyconfig template dictionary with easyconfig.TEMPLATE_NAMES_EASYBLOCK_RUN_STEP names"""
 
         for name in TEMPLATE_NAMES_EASYBLOCK_RUN_STEP:
-            self.cfg.template_values[name[0]] = str(getattr(self, name[0], None))
+            if name[0] == 'module_name_full':
+                self.cfg.template_values[name[0]] = ActiveMNS().det_full_module_name(self.cfg)
```

Comment:
You should be able to use `self.cfg.full_mod_name` here (no need to re-determine the module name)